Content.Server/Flash/FlashSystem.cs

```diff
@@ -165,7 +165,7 @@ public void FlashArea(EntityUid source, EntityUid? user, float range, float dura
             foreach (var entity in flashableEntities)
             {
                 // Check for unobstructed entities while ignoring the mobs with flashable components.
-                if (!_interactionSystem.InRangeUnobstructed(entity, mapPosition, range, CollisionGroup.Opaque, (e) => flashableEntities.Contains(e)))
+                if (!_interactionSystem.InRangeUnobstructed(entity, mapPosition, range, CollisionGroup.Opaque, (e) => flashableEntities.Contains(e) || e == source))
```

Comment:
Better optimisation would be to make it a static lambda but sometimes they're annoying to write.